apps/remote_control/lib/lexical/remote_control/project_node.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Lexical.RemoteControl.ProjectNode do
+  use GenServer
+
+  def start_link(options) do
+    GenServer.start_link(__MODULE__, options, [])
+  end
+
+  def init(options) do
+    wait_boot = options[:wait_boot] || 5_000
+    send(self(), :start_remote_control)
+    {:ok, %{options: options, wait_boot: wait_boot}}
+  end
+
+  def handle_info(:start_remote_control, state) do
+    options = state[:options]
+
+    cmd =
+      "elixir -pa #{options[:paths]} --name #{options[:name]} --cookie #{options[:cookie]} --no-halt -e 'Node.connect(#{inspect Node.self()})'"
```

Comment:
you'll likely need to use the function defined in `remote_control.ex` to find the elixir executable.